DCode.Data/TaskRepository/TaskRepository.cs

```diff
@@ -192,5 +192,10 @@ public IEnumerable<task> GetClientServiceTasksCreatedForDateRange(DateTime start
 
             return query.ToList();
         }
+        public string GetProficiencyfromId(int? proficiencyId)

```

Comment:
Please create a separate `IProficiencyRepository`. This should have two methods:
1) Pulling all the proficiency values.( Needed in one of the review comments ahead.)
2) Getting a proficiency value by ID.

Operations related Proficiency should not be present in repositories where they do not have any concern.